test/98_unittest.pm

```diff
@@ -289,6 +320,12 @@ sub UnitTest_mock_log3
   <ul><u>example:</u><br>
   <code>define test1 UnitTest dummyDuino ( { Log3 undef, 2, "this is a Log Message inside our Test";; } )
   </code></ul><br>
+  
+  <b>Attribute</b><br>
+	<ul><li><a name="disable"></a>disable<br>
+		This activates / deactivates the test. (0 = on | 1 = off)<br>
```

Comment:
A UnitTest definition can be disabled with the attribute disable. If disabled, the perl code provided in the definition will not be executed. The readings "test_output" and "test_failure" from this definition will be deleted.
If you delete this attribute or setting it to 0, the test will start immediatly